x-pack/solutions/security/plugins/security_solution/public/flyout/shared/components/preview_link.tsx

```diff
@@ -76,6 +84,15 @@ const getPreviewParams = (
         id: RulePreviewPanelKey,
         params: { ruleId, banner: RULE_PREVIEW_BANNER, isPreviewMode: true },
       };
+    case EVENT_SOURCE_FIELD_DESCRIPTOR:
+      return {
+        id: DocumentDetailsRightPanelKey,
+        params: {
+          id: value,
+          scopeId,
+          indexName: ancestorsIndexName,
```

Comment:
I forgot to notice this in my previous desk test, but the preview is missing the banner at the top. See the video below to show the difference between this preview and others:

https://github.com/user-attachments/assets/b4985572-4c21-4de8-b0af-7d1e0bf1031f

This is how I'm expecting the event preview banner to look like:
![Screenshot 2025-07-02 at 8 57 55 AM](https://github.com/user-attachments/assets/8006e185-308d-4caf-8f4d-c0b138d35348)

You should just have to add the following value to the `params`:
```typescript
banner: EVENT_PREVIEW_BANNER,
```
with `EVENT_PREVIEW_BANNER` imported from [here](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/preview/constants.ts#L18)

Reply:
Oh thanks for the catch! 😄 
I will fix it right away ☺️

Reply:
Fixed with the latest commit. Here's a screengrab of my desk test:

https://github.com/user-attachments/assets/f82c9dba-2e2e-4bbd-b8ca-cb893be2fd02